html_app/assignments/scb_ex/addmutlpledialog.soy

```diff
@@ -1,93 +0,0 @@
-{namespace scb_ex_old}
-
-/**
-@param template
-@param assignment
-@param experiment
-*/
-{template .dialog}
-<div class='scb_ex_inner_dialog' role='dialog' aria-label='Add Samples'>
```

Comment:
If you change the namespace here to scb_ex and in addmultipledioalog2.soy change the namespace to something else, and go to setup any of the NSF sample exercises, this older version of the dialog will be used.